common/dllserver/thorplugin.cpp

```diff
@@ -634,7 +634,7 @@ bool SafePluginMap::addPlugin(const char *path, const char *dllname)
         if (!dll)
         {
             Owned<PluginDll> n = new PluginDll(path, NULL);
-            if (!n->load(true, false) || !n->init(pluginCtx))
+            if (!n->load(false, false) || !n->init(pluginCtx))
```

Comment:
There would be potential implications if we had multiple plugins that shared a common library that was not linked in by the system - might end up with two copies of it - but I'm not aware that we do so anywhere. Regression suite all seems fine.

I can put in a comment explaining the decision.